package.json

```diff
@@ -3,11 +3,15 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.0.0",
+    "@fortawesome/fontawesome-free": "^5.15.3",
     "@rails/actioncable": "^6.0.0",
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.2.1",
     "babel-loader": "^8.2.2",
+    "bootstrap": "^4.6.0",
+    "jquery": "^3.6.0",
+    "popper.js": "^1.16.1",
```

Comment:
Why do you need to include` popper.js`?

Reply:
According to Bootstrap documentation is required but I read about the usage of `popper.js` and only is required mainly for tooltips, popovers, dropdowns, nor modals. So I removed this dependency because I don't use those features, and works fine.